python/feathub/processors/flink/tests/test_redis_client.py

```diff
@@ -107,12 +99,15 @@ def test_get_online_features(self):
             timestamp_field="ts",
         )
 
-        self.client.build_features([source, self.on_demand_feature_view])
+        on_demand_feature_view = self._get_on_demand_feature_view(
```

Comment:
Can you explain why `self.on_demand_feature_view` is replaced with `self._get_on_demand_feature_view(...)`?

Reply:
Sorry for the confusion. This is temporary code and should be removed.